packages/pwa-kit-dev/src/ssr/server/build-dev-server.js

```diff
@@ -172,11 +172,11 @@ export const DevServerMixin = {
                     namespace: 'DevServerMixin._setupExtensions',
                     additionalProperties: {error: e}
                 })
-                return
+                throw e
```

Comment:
🤔 this is the only time in `_setupExtensions` when an error is thrown. 

I believe the initial intention is to catch the errors such that
- an issue within 1 app extension will not affect the subsequent extensions
- all extensions are processed, regardless if there's an error in them or not

So I think that this line should be a `return` here.